build/steps-init.yml

```diff
@@ -28,8 +28,8 @@ steps:
 
 - script: |
     rustup install nightly
-    rustup component add rustfmt clippy
-    rustup component add rustfmt clippy --toolchain nightly
+    rustup component add rustfmt clippy llvm-tools-preview
+    rustup component add rustfmt clippy llvm-tools-preview --toolchain nightly
```

Comment:
(Just curiosity, feel free to ignore)
What will be lost if the green line 31 (L31) is removed?

Reply:
This is a quirk of rustup and cargo component installations. Rustup (the tool that handles installing Rust and related components) can install multiple "toolchains" (see https://rust-lang.github.io/rustup/concepts/toolchains.html), and components can be installed in each of those toolchains. The first part on L31 installs the components for the default toolchain, and the second line installs those same coponents for the "nightly" toolchain which includes additional Rust features that have not yet been fully stabilized in an official release.